src/pages/partners/affiliate-ib/_dmt5-cards.tsx

```diff
@@ -131,7 +181,7 @@ const CardText = styled(Text)`
 `
 const AccordionWrapper = styled.div`
     max-width: 99.6rem;
-    width: fit-content;
+    width: 100%;
```

Comment:
What is the different between fit-content and 100% width here?

Reply:
this has been changed to match the design due to the addition of a fourth element and the division of elements into 2 blocks, width 100% not work correctly with new approach